lib/generator/template_values.rb

```diff
@@ -1,24 +1,30 @@
 module Generator
   # Contains methods accessible to the ERB template
   class TemplateValues
-    attr_reader :abbreviated_commit_hash, :version, :test_cases
+    attr_reader :abbreviated_commit_hash, :version, :exercise_name, :test_cases
```

Comment:
Here is a wacky thought, do the ClassNames need to be ProblemSpecific?

Currently I think the answer is yes, but philosophically, I suspect not.